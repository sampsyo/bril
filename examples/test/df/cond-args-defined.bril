# ARGS: defined

@main(awesome_integer: int) {
.entry:
  print awesome_integer;
}
