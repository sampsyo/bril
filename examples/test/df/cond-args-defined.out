entry:
  in:  awesome_integer
  out: awesome_integer
